Tasks/KubernetesManifestV0/ThirdPartyNotices.txt

```diff
@@ -1621,7 +1621,7 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
 =========================================
-END OF VSTS-Task-Lib NOTICES, INFORMATION, AND LICENSE
+END OF Azure-Pipelines-Task-Lib NOTICES, INFORMATION, AND LICENSE
 
 %% wrappy NOTICES, INFORMATION, AND LICENSE BEGIN HERE
 =========================================
```

Comment:
I suppose the TPN file needs to be updated, as `utility-common` dependency is removed, could you check if there are references of modules here in the TPN, but not in the task?